src/amo/reducers/reviews.js

```diff
@@ -81,6 +85,9 @@ export type ReviewsState = {|
     // or null if one does not exist yet.
     [userIdAddonIdVersionId: string]: number | null,
   },
+  ratingSummary: {
+    [addonId: string]: RatingSummaryType,
```

Comment:
Why would `addonId` become a string here? It has been typed as `number` in `FetchRatingSummaryParams`.

Reply:
Whoops, I suppose it should be `number` then. Object keys can only be [strings or symbols](https://stackoverflow.com/questions/6066846/keys-in-javascript-objects-can-only-be-strings) (which we don't use) so Flow doesn't really care if we make it `number` or `string` here. It will always end up as a string. You're right, though; the input should always be a number.